Ficek_QuantiApp/fastlane/.env.default

```diff
@@ -0,0 +1,90 @@
+#General
+TEAM_ID = "2005235"
+APP_IDENTIFIER = "com.quanti.swift.NN-mobile-communicator"
```

Comment:
Tohle neni asi spravnej naming, to samy dole to APP_NAME atd. Nevim kde ste to vzali, pre sem se koukal do tech templates a tam to neni...